workflows/taxprofiler.nf

```diff
@@ -89,7 +89,7 @@ workflow TAXPROFILER {
     // MODULE: Run FastQC
     //
     FASTQC (
-        INPUT_CHECK.out.fastq
+        INPUT_CHECK.out.fastq.concat( INPUT_CHECK.out.nanopore )
```

Comment:
```suggestion
        INPUT_CHECK.out.fastq.mix( INPUT_CHECK.out.nanopore )
```

Mix is generally preferred, as then we don't need to wait for any other previous steps (that may ocur)